OpenRA.Mods.Common/Traits/Render/WithDecoration.cs

```diff
@@ -42,6 +45,9 @@ public class WithDecorationInfo : UpgradableTraitInfo, ITraitInfo
 		[Desc("Should this be visible to enemy players?")]
 		public readonly bool ShowToEnemies = false;
 
+		[Desc("Should this only be rendered when actor is in world?")]
+		public readonly bool OnlyWhenInWorld = true;
```

Comment:
Fixing paradrop (which has always been a bit of a hack) to either use a parachute actor and `WithCargo` or an `ActorPreview` would be two better options.